dataplug-gcalendar/app/com/hubofallthings/dataplugCalendar/apiInterfaces/GoogleCalendarsInterface.scala

```diff
@@ -20,7 +20,7 @@ import com.hubofallthings.dataplug.services.UserService
 import com.hubofallthings.dataplug.utils.{ AuthenticatedHatClient, FutureTransformations, Mailer }
 import com.hubofallthings.dataplugCalendar.models.GoogleCalendar
 import com.mohiva.play.silhouette.api.repositories.AuthInfoRepository
-import com.mohiva.play.silhouette.impl.providers.oauth2.GoogleProvider
+import com.hubofallthings.dataplugCalendar.apiInterfaces.authProviders._
```

Comment:
maybe `com.hubofallthings.dataplugCalendar.google.apiInterfaces.authProviders` instead of just Calendar?

Reply:
Sorry I didn't get that. ` com.hubofallthings.dataplugCalendar.apiInterfaces.authProviders._` is importing everything under authProviders. An alternative to make it more explicit will be ` com.hubofallthings.dataplugCalendar.apiInterfaces.authProviders.GoogleProvider._`

Reply:
Ya, just make it explicitly Google.  If we add a Zoho Calendar, or some other calendar, it will be cleaner.

Reply:
I support improved package namespacing with `google` package. @dataswifty can you create an issue for that?